amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -8,492 +8,967 @@ input AMPLIFY {
+  unlockable: Boolean
 }
 
+"""
+Represents the status of a Colony.
+
+This contains important meta information about the Colony's token and other fundamental settings
+"""
 type ColonyStatus {
+  """Status information for the Colony's native token."""
   nativeToken: NativeTokenStatus
-  recovery: Boolean # if it's in recovery mode
+  """Whether the Colony is in recovery mode."""
+  recovery: Boolean
 }
 
-# Applies to both Colonies Tokens and Events, but not all fields are revlant to all
-# It does not apply to user accounts as they can live on all networks
+"""
+Represents metadata related to a blockchain event.
+"""
 type ChainMetadata {
+  """The network the event occurred on."""
   network: Network
+  """The chain ID of the event."""
   chainId: Int
+  """The transaction hash of the event."""
   transactionHash: String
+  """The log index of the event."""
   logIndex: Int
+  """The block number of the event."""
   blockNumber: Int
 }
 
+"""
+Represents metadata for a user's profile. Mostly user specific settings.
+"""
 type ProfileMetadata {
+  """List of email permissions for the user."""
   emailPermissions: [String!]!
 }
 
+"""
+Represents a contributor within the Colony Network.
+
+A contributor is a Colony member who has reputation.
+"""
 type Contributor {
+  """User data associated with the contributor."""
   user: User
+  """Reputation percentage of the contributor (of all reputation within the Colony)."""
   reputationPercentage: String
+  """Reputation amount of the contributor (as an absolute number)."""
   reputationAmount: String
 }
 
+"""
+Represents a watcher within the Colony Network.
+
+A watcher is a Colony member who doesn't have reputation.
+"""
 type Watcher {
+  """User data associated with the watcher."""
   user: User
 }
 
+"""Represents an ERC20-compatible token that is used by Colonies and users."""
 type Token @model {
+  """Unique identifier for the token (contract address)."""
   id: ID!
     @index(name: "byAddress", queryField: "getTokenByAddress")
     @index(sortKeyFields: ["createdAt"]) # contract address
+  """Name of the token."""
   name: String!
+  """Symbol of the token."""
   symbol: String!
+  """Decimal precision of the token."""
   decimals: Int!
+  """Type of the token. See `TokenType` for more information"""
   type: TokenType @index(name: "byType", queryField: "getTokensByType")
+  """List of colonies using the token."""
   colonies: [Colony] @manyToMany(relationName: "ColonyTokens")
+  """List of users using the token."""
   users: [User] @manyToMany(relationName: "UserTokens")
+  """URL of the token's avatar image (logo)."""
   avatar: String
+  """URL of the token's thumbnail image (Small logo)."""
   thumbnail: String
+  """Metadata related to the chain of the token."""
   chainMetadata: ChainMetadata
+  """Timestamp of the token model's creation in the database."""
   createdAt: AWSDateTime!
 }
 
+"""
+**Deprecated** Type for a unique identfier for a Colony
+"""
 type ColonyID {
```

Comment:
If this isn't being used, we could just remove?

Reply:
+1, we can remove it as discussed in the call